ktor-server/ktor-server-jetty/jvm/src/io/ktor/server/jetty/JettyKtorHandler.kt

```diff
@@ -72,7 +71,7 @@ internal class JettyKtorHandler(
     ) {
         try {
             val contentType = request.contentType
-            if (contentType != null && contentType.startsWith("multipart/")) {
+            if (contentType != null && contentType.startsWith("multipart/", ignoreCase = true)) {
```

Comment:
I would extract constant here

Reply:
I could add a constant `ContentType.Application.TYPE`. I also thought about adding an operator function `contains`:

```kotlin
public object ContentType {
    public object Application {
        // ...

        public operator fun contains(contentType: CharSequence): Boolean =
            contentType.startsWith("$APPLICATION/", ignoreCase = true)
    }
}
```
And then check the content type using `in`: 
```kotlin
if (contentType in ContentType.Application) { ... }
```
What do you think?

Reply:
I'll make it in a separate PR

Reply:
@bjhham, what do you think?